---
layout: papers-publications
permalink: /papers-publications/

# Banner Content
# =================================
title: Papers and Publications
subtitle: A mix of papers that were published by contributors to the Digital Slide Archive and by unaffiliated researchers who have found the system to be useful in their work.
hero_image: assets/img/home-callouts/papers_publications.jpg

# Papers and Publications list
# =================================

[//]: # (keys are title, full-citation, DOI_link, link, DOI, PMID)

publications:
  - title: "FUSION: A Web-Based Application for in-Depth Exploration of Multi-Omics Data with Brightfield Histology."
    full-citation: "Border, Samuel P., Ricardo Melo Ferreira, Nicholas Lucarelli, Suhas Katari Chaluva Kumar, Anindya S. Paul, David Manthey, Laura Barisoni, et al. 2025. “FUSION: A Web-Based Application for in-Depth Exploration of Multi-Omics Data with Brightfield Histology.” Nature Communications 16 (1)."
    DOI_link: https://doi.org/10.1038/s41467-025-63050-9
    link: https://www.nature.com/articles/s41467-025-63050-9
    DOI: 10.1038/s41467-025-63050-9
    PMID: 40998789
  - title: "HistomicsTK: A Python Toolkit for Pathology Image Analysis Algorithms."
    full-citation: "Pourakpour, Fattaneh, Ákos Szölgyén, Ramin Nateghi, David A Gutman, David Manthey, and Lee AD Cooper. 2025. “HistomicsTK: A Python Toolkit for Pathology Image Analysis Algorithms.” SoftwareX 31 (September): 102318."
    DOI_link: https://doi.org/10.1016/j.softx.2025.102318
    link: https://doi.org/10.1016/j.softx.2025.102318
    DOI: 10.1016/j.softx.2025.102318
  - title: "ComPRePS 2.0: Enabling Massive-Scale Distributed Computing on High-Performance Computing Cluster for Histopathological Data Processing."
    full-citation: "Katari Chaluva Kumar, Suhas, Anindya S. Paul, Haitham Abdelazim, Will Dunklin, David Manthey, Oleksandr Moskalenko, Sayat Mimar, Erik Deumens, and Pinaki Sarder. 2025. “ComPRePS 2.0: Enabling Massive-Scale Distributed Computing on High-Performance Computing Cluster for Histopathological Data Processing.” Edited by John E. Tomaszewski and Aaron D. Ward. Medical Imaging 2025: Digital and Computational Pathology. SPIE."
    DOI_link: https://doi.org/10.1117/12.3048195
    link: https://doi.org/10.1117/12.3048195
    DOI: 10.1117/12.3048195
  - title: "Efficiently Writing and Visualizing Many-Dimension Large Images."
    full-citation: "Haley, Anne, David Manthey, Lee Cooper, and David Gutman. 2024. “Poster Presentations: Efficiently Writing and Visualizing Many-Dimension Large Images.” Digital Pathology & AI Congress, May 7-8, 2024."
    link: ../assets/pdfs/Poster2024DigPathAI.pdf
    link_text: Poster2024DigPathAI.pdf
  - title: "A User-Friendly Tool for Cloud-Based Whole Slide Image Segmentation with Examples from Renal Histopathology."
    full-citation: "Lutnick, Brendon, David Manthey, Jan U. Becker, Brandon Ginley, Katharina Moos, Jonathan E. Zuckerman, Luis Rodrigues, et al. 2022. “A User-Friendly Tool for Cloud-Based Whole Slide Image Segmentation with Examples from Renal Histopathology.” Communications Medicine. Springer Science and Business Media LLC."
    DOI_link: https://doi.org/10.1038/s43856-022-00138-z
    link: https://doi.org/10.1038/s43856-022-00138-z
    PMID: 35996627
    DOI: 10.1038/s43856-022-00138-z
  - title: "PodoCount: A Robust, Fully Automated, Whole-Slide Podocyte Quantification Tool."
    full-citation: "Santo, Briana A., Darshana Govind, Parnaz Daneshpajouhnejad, Xiaoping Yang, Xiaoxin X. Wang, Komuraiah Myakala, Bryce A. Jones, et al. 2022. “PodoCount: A Robust, Fully Automated, Whole-Slide Podocyte Quantification Tool.” Kidney International Reports. Elsevier BV."
    DOI_link: https://doi.org/10.1016/j.ekir.2022.03.004
    link: https://doi.org/10.1016/j.ekir.2022.03.004
    PMID: 35694561
    DOI: 10.1016/j.ekir.2022.03.004
  - title: "A Tool for Federated Training of Segmentation Models on Whole Slide Images."
    full-citation: "Lutnick, Brendon, David Manthey, Jan U. Becker, Jonathan E. Zuckerman, Luis Rodrigues, Kuang-Yu Jen, and Pinaki Sarder. 2022. “A Tool for Federated Training of Segmentation Models on Whole Slide Images.” Journal of Pathology Informatics. Elsevier BV."
    DOI_link: https://doi.org/10.1016/j.jpi.2022.100101
    link: https://doi.org/10.1016/j.jpi.2022.100101
    PMID: 35910077
    DOI: 10.1016/j.jpi.2022.100101
  - title: "Explainable Nucleus Classification Using Decision Tree Approximation of Learned Embeddings"
    full-citation: "Amgad, Mohamed, LameesA Atteya, Hagar Hussein, Kareem Hosny Mohammed, Ehab Hafiz, Maha A T Elsebaie, Pooya Mobadersany, et al. 2021. “Explainable Nucleus Classification Using Decision Tree Approximation of Learned Embeddings.” Edited by Jinbo Xu. Bioinformatics, September."
    DOI_link: https://doi.org/10.1093/bioinformatics/btab670
    link: https://doi.org/10.1093/bioinformatics/btab670
    PMID: 34586355
    DOI: 10.1093/bioinformatics/btab670
  - title: "PodoSighter: A Cloud-Based Tool for Label-Free Podocyte Detection in Kidney Whole Slide Images"
    full-citation: "Govind, Darshana, Jan Becker, Jeffrey Miecznikowski, Avi Rosenberg, Julien Dang, Pierre Louis Tharaux, Rabi Yacoub, et al. 2021. “PodoSighter: A Cloud-Based Tool for Label-Free Podocyte Detection in Kidney Whole Slide Images.” Journal of the American Society of Nephrology, September, ASN.2021050630."
    DOI_link: https://doi.org/10.1681/asn.2021050630
    link: https://doi.org/10.1681/asn.2021050630
    PMID: 34479966
    DOI: 10.1681/asn.2021050630
  - title: User Friendly, Cloud Based, Whole Slide Image Segmentation.
    full-citation: "Lutnick, Brendon, Avinash Kammardi Shashiprakash, David Manthey, and Pinaki Sarder. 2021. “User Friendly, Cloud Based, Whole Slide Image Segmentation.” In Medical Imaging 2021: Digital Pathology, edited by John E. Tomaszewski and Aaron D. Ward. SPIE."
    DOI_link: https://doi.org/10.1117/12.2581383
    link: https://doi.org/10.1117/12.2581383
    PMID: 34366542
    DOI: 10.1117/12.2581383
  - title: A Distributed System Improves Inter-Observer and AI Concordance in Annotating Interstitial Fibrosis and Tubular Atrophy.
    full-citation: "Kammardi Shashiprakash, Avinash, Brendon Lutnick, Brandon Ginley, Darshana Govind, Nicholas Lucarelli, Kuang-Yu Jen, Avi Z. Rosenberg, et al. 2021. “A Distributed System Improves Inter-Observer and AI Concordance in Annotating Interstitial Fibrosis and Tubular Atrophy.” In Medical Imaging 2021: Digital Pathology, edited by John E. Tomaszewski and Aaron D. Ward. SPIE."
    DOI_link: https://doi.org/10.1117/12.2581789
    link: https://doi.org/10.1117/12.2581789
    PMID: 34366540
    DOI: 10.1117/12.2581789
  - title: Machine-based detection and classification for bone marrow aspirate differential counts - initial development focusing on nonneoplastic cells.
    full-citation: "Chandradevan, Ramraj, Ahmed A. Aljudi, Bradley R. Drumheller, Nilakshan Kunananthaseelan, Mohamed Amgad, David A. Gutman, Lee A. D. Cooper, and David L. Jaye. “Machine-Based Detection and Classification for Bone Marrow Aspirate Differential Counts: Initial Development Focusing on Nonneoplastic Cells.” Laboratory Investigation 100, no. 1 (September 30, 2019): 98–109."
    DOI_link: https://doi.org/10.1038/s41374-019-0325-7
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=31570774
    PMID: 31570774
    DOI: 10.1038/s41374-019-0325-7
  - title: Phosphoinositide 3-Kinase Signaling Can Modulate MHC Class I and II Expression.
    full-citation: "Chandrasekaran, Sanjay, Maiko Sasaki, Christopher D. Scharer, Haydn T. Kissick, Dillon G. Patterson, Kelly R. Magliocca, John T. Seykora, et al. “Phosphoinositide 3-Kinase Signaling Can Modulate MHC Class I and II Expression.” Molecular Cancer Research 17, no. 12 (September 23, 2019): 2395–2409."
    DOI_link: https://doi.org/10.1158/1541-7786.mcr-19-0545
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=31548239
    PMID: 31548239
    DOI: 10.1158/1541-7786.MCR-19-0545
  - title: Structured crowdsourcing enables convolutional segmentation of histology images.
    full-citation: "Amgad, Mohamed, Habiba Elfandy, Hagar Hussein, Lamees A Atteya, Mai A T Elsebaie, Lamia S Abo Elnasr, Rokia A Sakr, et al. “Structured Crowdsourcing Enables Convolutional Segmentation of Histology Images.” Edited by Robert Murphy. Bioinformatics 35, no. 18 (February 6, 2019): 3461–67."
    DOI_link: https://doi.org/10.1093/bioinformatics/btz083
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=30726865
    PMID: 30726865
    DOI: 10.1093/bioinformatics/btz083
  - title: Multi-faceted computational assessment of risk and progression in oligodendroglioma implicates NOTCH and PI3K pathways.
    full-citation: "Halani, Sameer H., Safoora Yousefi, Jose Velazquez Vega, Michael R. Rossi, Zheng Zhao, Fatemeh Amrollahi, Chad A. Holder, et al. “Multi-Faceted Computational Assessment of Risk and Progression in Oligodendroglioma Implicates NOTCH and PI3K Pathways.” Npj Precision Oncology 2, no. 1 (November 6, 2018)."
    DOI_link: https://doi.org/10.1038/s41698-018-0067-9
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=30417117
    PMID: 30417117
    DOI: 10.1038/s41698-018-0067-9
  - title: PanCancer insights from The Cancer Genome Atlas - the pathologist's perspective.
    full-citation: "Cooper, Lee AD, Elizabeth G Demicco, Joel H Saltz, Reid T Powell, Arvind Rao, and Alexander J Lazar. “PanCancer Insights from The Cancer Genome Atlas: The Pathologist’s Perspective.” The Journal of Pathology 244, no. 5 (February 22, 2018): 512–24."
    DOI_link: https://doi.org/10.1002/path.5028
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=29288495
    PMID: 29288495
    DOI: 10.1002/path.5028
  - title: Predicting cancer outcomes from histology and genomics using convolutional networks.
    full-citation: "Mobadersany, Pooya, Safoora Yousefi, Mohamed Amgad, David A. Gutman, Jill S. Barnholtz-Sloan, José E. Velázquez Vega, Daniel J. Brat, and Lee A. D. Cooper. “Predicting Cancer Outcomes from Histology and Genomics Using Convolutional Networks.” Proceedings of the National Academy of Sciences 115, no. 13 (March 12, 2018): E2970–79."
    DOI_link: https://doi.org/10.1073/pnas.1717139115
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=29531073
    PMID: 29531073
    DOI: 10.1073/pnas.1717139115
  - title: Informatics Approaches to Address New Challenges in the Classification of Lymphoid Malignancies.
    full-citation: "Jordan, Jacob, Jordan S. Goldstein, David L. Jaye, Metin Gurcan, Christopher R. Flowers, and Lee A.D. Cooper. “Informatics Approaches to Address New Challenges in the Classification of Lymphoid Malignancies.” JCO Clinical Cancer Informatics, no. 2 (December 2018): 1–9."
    DOI_link: https://doi.org/10.1200/cci.17.00039
    link: https://www.ncbi.nlm.nih.gov/pubmed/30637363
    PMID: 30637363
    DOI: 10.1200/CCI.17.00039
  - title: Comprehensive and Integrated Genomic Characterization of Adult Soft Tissue Sarcomas.
    full-citation: "Abeshouse, Adam, Clement Adebamowo, Sally N. Adebamowo, Rehan Akbani, Teniola Akeredolu, Adrian Ally, Matthew L. Anderson, et al. “Comprehensive and Integrated Genomic Characterization of Adult Soft Tissue Sarcomas.” Cell 171, no. 4 (November 2017): 950–965.e28."
    DOI_link: https://doi.org/10.1016/j.cell.2017.10.014
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=29100075
    PMID: 29100075
    DOI: 10.1016/j.cell.2017.10.014
  - title: The Digital Slide Archive - A Software Platform for Management, Integration, and Analysis of Histology for Cancer Research.
    full-citation: "Gutman, David A., Mohammed Khalilia, Sanghoon Lee, Michael Nalisnik, Zach Mullen, Jonathan Beezley, Deepak R. Chittajallu, David Manthey, and Lee A.D. Cooper. “The Digital Slide Archive: A Software Platform for Management, Integration, and Analysis of Histology for Cancer Research.” Cancer Research 77, no. 21 (October 31, 2017): e75–78."
    DOI_link: https://doi.org/10.1158/0008-5472.can-17-0629
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=29092945
    PMID: 29092945
    DOI: 10.1158/0008-5472.CAN-17-0629
  - title: Molecular Profiling Reveals Biologically Discrete Subsets and Pathways of Progression in Diffuse Glioma.
    full-citation: "Ceccarelli, Michele, Floris P. Barthel, Tathiane M. Malta, Thais S. Sabedot, Sofie R. Salama, Bradley A. Murray, Olena Morozova, et al. “Molecular Profiling Reveals Biologically Discrete Subsets and Pathways of Progression in Diffuse Glioma.” Cell 164, no. 3 (January 2016): 550–63."
    DOI_link: https://doi.org/10.1016/j.cell.2015.12.028
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=26824661
    PMID: 26824661
    DOI: 10.1016/j.cell.2015.12.028
  - title: Novel genotype-phenotype associations in human cancers enabled by advanced molecular platforms and computational analysis of whole slide images.
    full-citation: "Cooper, Lee AD, Jun Kong, David A Gutman, William D Dunn, Michael Nalisnik, and Daniel J Brat. “Novel Genotype-Phenotype Associations in Human Cancers Enabled by Advanced Molecular Platforms and Computational Analysis of Whole Slide Images.” Laboratory Investigation 95, no. 4 (January 19, 2015): 366–76."
    DOI_link: https://doi.org/10.1038/labinvest.2014.153
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=25599536
    PMID: 25599536
    DOI: 10.1038/labinvest.2014.153
  - title: Cancer Digital Slide Archive - an informatics resource to support integrated in silico analysis of TCGA pathology data.
    full-citation: "Gutman, David A, Jake Cobb, Dhananjaya Somanna, Yuna Park, Fusheng Wang, Tahsin Kurc, Joel H Saltz, Daniel J Brat, Lee A D Cooper, and Jun Kong. “Cancer Digital Slide Archive: An Informatics Resource to Support Integrated in Silico Analysis of TCGA Pathology Data.” Journal of the American Medical Informatics Association 20, no. 6 (November 2013): 1091–98."
    DOI_link: https://doi.org/10.1136/amiajnl-2012-001469
    link: https://www.ncbi.nlm.nih.gov/pubmed/?term=23893318
    PMID: 23893318
    DOI: 10.1136/amiajnl-2012-001469
---
